### PR TITLE
docs update for windows and arc windows server control

### PR DESCRIPTION
**Pull request by pranchalsomani**
Added docs for Azure_HybridCompute_DP_Use_Secure_TLS_Version_Trial and Azure_VirtualMachine_DP_Use_Secure_TLS_Version_Trial
 control

### CODE REVIEW

File: Control coverage/Feature/HybridCompute.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri//providers/Microsoft.Authorization/policyDefinitions/828ba269-bf7f-4082-83dd-633417bc391d) 
+	 "/providers/Microsoft.Authorization/policyDefinitions/af6cd1bd-1635-48cb-bde7-5b15693900b9"
+
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+-->
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list Hybrid Networks at
```
Comment (Aboli-msft) [inline Control coverage/Feature/HybridCompute.md:71]: Should it be API to list Hybrid compute servers?

**Review by Aboli-msft — APPROVED** [review]

File: Control coverage/Feature/HybridCompute.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ "ApplicableOsTypes": [
+          "Windows"
+  ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> Azure Policy Configure secure communication protocols (TLS 1.1 or TLS 1.2) is set compliant state to “Compliant”.
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/HybridCompute.md:33]: Can we change the grammar for this line. Maybe add quotes for "Configure secure communication protocols".
The way the current line is, it's not making much sense when I'm trying to read it.

File: Control coverage/Feature/HybridCompute.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+>
+> **Failed:** 
+>  Azure Policy Configure secure communication protocols (TLS 1.1 or TLS 1.2) is set compliant state to “Non-Compliant”.
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/HybridCompute.md:36]: Same as above.

File: Control coverage/Feature/HybridCompute.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	 "/providers/Microsoft.Authorization/policyDefinitions/af6cd1bd-1635-48cb-bde7-5b15693900b9"
+
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+-->
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list hybrid machines at
+subscription level:
+/subscriptions/{subscriptionId}/providers/Microsoft.HybridCompute/machines?api-version=2022-08-11-preview<br />
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/HybridCompute.md:73]: Just a minor check - Can we try adding hyperlinks for these as well?
For the new ones we create anyway.

For example, this for API:
https://learn.microsoft.com/en-us/rest/api/hybridcompute/machines/list-by-subscription?tabs=HTTP

File: Control coverage/Feature/HybridCompute.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+**Properties:** properties.osType
+
+- Azure Policy used for evaluation: /providers/Microsoft.Authorization/policyDefinitions/828ba269-bf7f-4082-83dd-633417bc391d
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/HybridCompute.md:76]: Same as above.

Somewhere from here? https://learn.microsoft.com/en-us/azure/governance/policy/samples/built-in-policies
Just check if an appropriate link can be given.

File: Control coverage/Feature/VirtualMachine.md [hunk 6]
```diff
@@ -1385,3 +1386,73 @@ Open remote management ports expose a VM/compute node to a high level of risk fr
 
 ___ 
 
+## Azure_VirtualMachine_DP_Use_Secure_TLS_Version_Trial
```
Comment (aakashpahwa-msft) [inline Control coverage/Feature/VirtualMachine.md:1389]: Can you check the comments for HybridCompute for VirtualMachine changes as well.

Comment (aakashpahwa-msft) [reply Control coverage/Feature/VirtualMachine.md:1389]: I think all the comments are applicable for VirtualMachine changes as well.

**Review by aakashpahwa-msft — APPROVED** [review]
LGTM

**Pull request merged**